mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -2812,7 +2823,60 @@ func versionsEqual(old, new string) (bool, error) {
 	return o.Equal(n), nil
 }
 
-func isComposer3(d *schema.ResourceData, config *transport_tpg.Config)	bool {
-	image_version := d.Get("config.0.software_config.0.image_version").(string)
-	return strings.Contains(image_version, "composer-3")
+func isComposer3(v interface{})	bool {
```

Comment:
This seems like it introduces a level of indirection that might be confusing in the long run, since each type of call basically only happens once - what would you think about making this always take a string, and passing in the appropriate value from the caller? Might be clearer?

Reply:
done